fonts.css

```diff
@@ -0,0 +1,112 @@
+@font-face {
+    font-family: 'PT Serif';
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/tni-htmlcss2.0/pullRequest?prid=6165338)